### ISSUE TITLE
[$40][MSFT-214] [Challenge] Date picker is not being highlighted with the navigation keys

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2876]
**Steps to Reproduce**
1. Go to the link https://www.topcoder.com/challenges
2. Click on filter button. 
3. Fill the Required fields with the tab function. 
4. Go to the Date Picker using the tab keys, it opens the date picker window. 
5. Use the arrow keys to select the dates as preferred. 
6. Hit enter and observe the date highlighted .

**Expected Result**
The User should be able to pick the date with the navigation keys.

**Actual Result**

User is able to select the date however, it's not being focused or highlighted with the navigation keys.

**Device**: Desktop/Labtop
**Operating System**: macOS
**Browser**: 

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.4.3 Focus Order
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=2d6767b3%2D0a20%2D473c%2D8e2b%2D337aeb296696&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FCreeya%2FBug%2D%20Date%20selection

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096169 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096169 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by r0hit-gupta** [comment]
Please assign me

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096169 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2876 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2876 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097059 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097059 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by drasticdpk** [comment]
Operating System: macOS
Browser: Chrome/Safari
URL: https://test-community-app.topcoder-dev.com/challenges

**Comment by drasticdpk** [comment]
Verified on 
MAC -Chrome/Safari
Windows - Chrome/Edge

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097059<br/><br/>```This is an automated message for crazyk via Topcoder X```
